yahoo_fantasy_api/league.py

```diff
@@ -733,6 +733,35 @@ def draft_results(self):
             dres.append(p)
         return(dres)
 
+    def transactions(self, tran_types, count):
+        '''
+        Fetch transactions of a given type for the league.
+
+        :param tran_types: The comman seperated types of transactions retrieve.  Valid values
+        are: add,drop,commish,trade
```

Comment:
This need to indented as it messes up the docs.  sphinx doesn't know that this line is a continuation of the `tran_types` parameter.  Similar comment for `count` parameter.